packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { AccountStatusResponse, GetAccountStatus } from '@deriv/api-types';
+import { Button, Loading } from '@deriv/components';
+import { WS, getPlatformRedirect, platforms } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import Expired from '../../../Components/poa/status/expired';
+import NeedsReview from '../../../Components/poa/status/needs-review';
+import NotRequired from '../../../Components/poa/status/not-required';
+import PoaStatusCodes from '../../../Components/poa/status/status-codes';
+import ProofOfAddressForm from './proof-of-address-form';
+import Submitted from '../../../Components/poa/status/submitted';
+import Unverified from '../../../Components/poa/status/unverified';
+import Verified from '../../../Components/poa/status/verified';
+import { populateVerificationStatus } from '../Helpers/verification.js';
+
+type TAuthenticationStatus = Record<
+    | 'allow_document_upload'
+    | 'allow_poi_resubmission'
+    | 'allow_poa_resubmission'
+    | 'is_age_verified'
+    | 'has_poi'
+    | 'has_submitted_poa'
+    | 'needs_poa'
+    | 'needs_poi'
+    | 'poa_address_mismatch'
+    | 'resubmit_poa',
+    boolean
+> & { document_status?: DeepRequired<GetAccountStatus>['authentication']['document']['status'] };
+
+const ProofOfAddressContainer = observer(() => {
+    const [is_loading, setIsLoading] = React.useState(true);
+    const [authentication_status, setAuthenticationStatus] = React.useState<TAuthenticationStatus>({
+        allow_document_upload: false,
+        allow_poi_resubmission: false,
+        allow_poa_resubmission: false,
+        needs_poi: false,
+        needs_poa: false,
+        has_poi: false,
+        resubmit_poa: false,
+        has_submitted_poa: false,
+        document_status: undefined,
+        is_age_verified: false,
+        poa_address_mismatch: false,
+    });
+
+    const { client, notifications, common } = useStore();
+    const { app_routing_history } = common;
+    const { landing_company_shortcode, has_restricted_mt5_account, is_switching } = client;
+    const { refreshNotifications } = notifications;
+
+    const is_mx_mlt = landing_company_shortcode === 'iom' || landing_company_shortcode === 'malta';
```

Comment:
why do we need this?

Reply:
@amina-deriv, I'm not sure, I just left it as it was before in the code

Reply:
The PR is already bloated. Can we make the refactoring of existing code in a separate Card?